requirements-dev.txt

```diff
@@ -9,3 +9,4 @@ mkdocs~=1.1
 mkdocs-material~=7.3
 mkdocs-material-extensions~=1.0
 mkdocstrings[python]~=0.18
+fcsparser~=0.2
```

Comment:
Where are we still using this? Tests?

Reply:
tests; I'll go ahead and change those tests though

Reply:
The last commit `deps(fcsparser): Remove fcsparser` can be squashed into `deps: Replace fcsparser with flowio`.